analyzer/server.go

```diff
@@ -207,6 +221,12 @@ func (s *Server) RegisterRpcEndpoints() {
 			"/rpc/flows",
 			s.FlowSearch,
 		},
+		{
+			"LayerConversation",
+			"GET",
+			"/data/conversation.json/{layer}",
```

Comment:
should be /rpc/flow/{layer}/conversation.json or something like that, meaning having the extension at the end